### PR TITLE
Basic vision simulation in autonomous

### PR DESCRIPTION
**Pull request by james-ward**

### CODE REVIEW

**Comment by james-ward** [comment]
If only I could get a basic kinematic model running so that the chassis and turret actually move...

**Comment by james-ward** [comment]
Now depends on #151

File: components/vision.py [hunk 1]
```diff
@@ -29,8 +37,36 @@ class Vision:
...
+                Transform2d(0, 0, 0),
+                self.CAMERA_HEIGHT,
+                10.0,
+                1280,
+                720,
+                0.0,
+            )
+            self.sim_vision_system.addSimVisionTarget(
+                SimVisionTarget(
+                    Pose2d(0, 0, 0), self.TARGET_HEIGHT, self.GOAL_RADIUS * 2, 0.05
```
Comment (auscompgeek) [inline components/vision.py:65]: It'd be worth using keyword arguments for the constants here, so it's clearer what they are.

**Review by auscompgeek — APPROVED** [review]
LGTM, just one minor comment

**Pull request merged**